libs/ledgerjs/packages/react-native-hw-transport-ble/src/BleTransport.ts

```diff
@@ -408,6 +412,7 @@ export default class BluetoothTransport extends Transport {
   static disconnect = async (id: any) => {
     log("ble-verbose", `user disconnect(${id})`);
     await bleManager.cancelDeviceConnection(id);
+    await delay(1000);
```

Comment:
Everything clear except this delay ? Maybe add a comment to explain why this delay here ?

Reply:
It was a test to see if introducing a 1s delay between a disconnect and the next potential connection improved the situation of our BLE stability. I'd rather leave it there for now, and indeed I should've added a comment. So I just did